### ISSUE TITLE
Possible problem with GT

### ISSUE THREAD

**Issue by AntiLibrary5** [SeasonDepth/SeasonDepth#7]
### Prediction on a sample on SeasonDepth_testset

Thank you authors. I wanted to roughly gauge the dataset and my concern is GT seems erroneous qualitatively. I understand the GT was created from the typical SfM pipeline. Am I missing something in the post-processing? Concern being error metrics won't be reflective of the model performance. Its similar for many other images (after browsing through). Do you agree?

![image](https://user-images.githubusercontent.com/48105310/161697664-cc31e144-b370-4729-99db-83541cf1aea1.png)

**Comment by yangbaoquan** [comment]
Thank you for using SeasonDepth.

That's right. Since the ground truth is generated from SfM and post processing, there may be some noise in images. To make the benchmark result more convincing, we found out the main source of noise and then masked them automatically. 

However, some specific noise, which may influence the final result badly, can only appear in some certain images, so we attempted to mask them individually. Due to the large number of images, some residues that won't affect overall performance are left, just like what you have mentioned in the picture. According to our experiments, while evaluating which method is better, the residues are negligible. 

Hope that can dismiss your concern.
